common/app/conf/switches/ABTestSwitches.scala

```diff
@@ -40,7 +40,7 @@ trait ABTestSwitches {
     "Test various margins at which ads are lazily-loaded in order to find the optimal one",
     owners = Seq(Owner.withGithub("zekehuntergreen")),
     safeState = Off,
-    sellByDate = Some(LocalDate.of(2022, 4, 14)),
+    sellByDate = Some(LocalDate.of(2022, 5, 1)),
```

Comment:
Think this needs to be a weekday :)

Reply:
🤦 how annoying. In protest I've updated it to the next day which is a bank holiday.